README.md

```diff
@@ -78,10 +78,12 @@ the package.
 |-----------------|---|
 |2.0|required|
 |3.0|required|
+|4.0|required|
 
 Every package in Universe must have a `package.json` file which specifies the high level metadata about the package.
 
-Currently, a package can specify one of two values for `.packagingVersion` either `2.0` or `3.0`; which version is declared
+Currently, a package can specify one of three values for `.packagingVersion`
+either `2.0` or `3.0` or `4.0`; which version is declared
```

Comment:
Let's motivate people to write version 4.0 packages. Let's replace the example below with a version 4.0 example.